solutions.md

```diff
@@ -60,9 +60,29 @@ clean :
 > 3. only .dat files recreated
 > 4. only analysis.tar.gz recreated
 
+~~~ 
```

Comment:
I don't think this should be here? The entire answer section is rendering as a code-style block.